README.md

```diff
@@ -43,188 +43,98 @@ Visit https://www.periculum.io/documentation/insights/#authenticationrequest for
-| `token` | `String` | **required**  [Get token](https://github.com/SteveOye/periculum-flutter/blob/docs_refactor/README.md#authentication) |
-| `statementKey` | `String` | **required** |
-
-
-```dart
-CreditScoreResponse response; //returns an CreditScoreResponse object 
-
-try {
-  response =
-    await FlutterPericulum.generateCreditScore(
-    token: "" //get token from authentication,
-    statementKey: '125',
-  );
-} catch (e) {
-   //handle exceptions
-}
+| `publicKey` | `String` | **required**  |
+| `phoneNumber` | `String` | **required** |
+| `bvn` | `String` | **required** |
+
+Returns a list of key 
+
+```dart 
+var flutterPericulum =
+          await FlutterPericulum.generateMobileAnalysis(
+        publicKey: 'publicKey',
+        bvn: '1234567890123',
+        phoneNumber: '090********',
+      );
 ```
 
-## Get Existing Credit Score
-
-###### 
-| Parameter | Type     | Description                |
-| :-------- | :------- | :------------------------- |
-| `token` | `String` | **required** [Get token](https://github.com/SteveOye/periculum-flutter/blob/docs_refactor/README.md#authentication) |
-| `statementKey` | `String` | **required** |
+## Generate Mobile Analysis for Version 1 of the SDK
 
-```dart
-List<CreditScoreResponse> response; //returns an List<CreditScoreResponse> object 
-
-try {
-  response =
-    await FlutterPericulum.getExisitingCreditScore(
-    token: "", //get token from authentication 
-    statementKey: '125',
-  );
-} catch (e) {
-   //handle exceptions
-}
-```
-## Get Statement Transactions
+Returns a Json String contain //{ "mobileInsightsOverviewKey": 6}
 ###### 
 | Parameter | Type     | Description                |
 | :-------- | :------- | :------------------------- |
-| `token` | `String` | **required**  [Get token](https://github.com/SteveOye/periculum-flutter/blob/docs_refactor/README.md#authentication) |
-| `statementKey` | `String` | **required** |
-
-
-```dart
-List<Transaction> response; //returns an List<Transaction> object 
-
-try {
-  response =
-    await FlutterPericulum.getStatementTransaction(
-    token: "" //get token from authentication,
-    statementKey: '125',
-  );
-} catch (e) {
-   //handle exceptions
-}
+| `publicKey` | `String` | **required**  |
+| `phoneNumber` | `String` | **required** |
+| `bvn` | `String` | **required** |
+
+```dart 
+var flutterPericulum =
+          await FlutterPericulum.generateMobileInsightV2(
+        publicKey: 'publicKey',
+        bvn: '1234567890123',
+        phoneNumber: '090********',
+      );
 ```
 
-## Get Existing Statement Analytics
-###### 
-| Parameter | Type     | Description                |
-| :-------- | :------- | :------------------------- |
-| `token` | `String` | **required**  [Get token](https://github.com/SteveOye/periculum-flutter/blob/docs_refactor/README.md#authentication) |
-| `statementKey` | `String` | **required** |
-
+## Update an existing Mobile Analysis for Version 2 of the SDK
```

Comment:
You also need to document the Generate Analysis for Version 1 of the SDK @SteveOye